tests/pytest/core/test_views.py

```diff
@@ -46,37 +47,29 @@ def test_form_valid(self, view, model_TransitAgency):
 
 
 @pytest.mark.django_db
-def test_agency_index_single_flow(mocker, model_TransitAgency, client, session_reset_spy, mocked_session_update):
-    mocker.patch("benefits.core.models.TransitAgency.all_active", return_value=[model_TransitAgency])
-
-    response = client.get(model_TransitAgency.index_url)
-
-    session_reset_spy.assert_called_once()
-    mocked_session_update.assert_called_once()
-
-    assert response.status_code == 200
-    assert response.template_name == "core/index--agency.html"
+class TestAgencyIndexView:
+    @pytest.fixture
+    def view(self, app_request, model_TransitAgency):
+        v = views.AgencyIndexView()
+        converter = TransitAgencyPathConverter()
+        v.setup(app_request, agency=converter.to_python(model_TransitAgency.slug))
+        return v
 
+    def test_view(self, view):
+        assert view.template_name == "core/index--agency.html"
 
-@pytest.mark.django_db
-def test_agency_index_multiple_flow(
```

Comment:
If my spelunking is correct, it looks like this test was left over from a time where an agency with only one enrollment flow would redirect from its agency index to `/eligibility/start/`, but let me know if I got that wrong and this is still a scenario that needs to be tested.

Reply:
Hmm I honestly can't remember. I know we have some shortcut redirect tricks like that for e.g. `/<slug>/eligibility` and `/<slug>/agency-card`, but I think you are right that this was an outdated test that doesn't need to be represented in the new structure.